scripts/install/install_unix.sh

```diff
@@ -427,24 +427,11 @@ set_package_type()
 set_download_urls()
 {
   if [ -z "$url" ] ; then
-    if [ -z "$version" ] ; then
-      # shellcheck disable=SC2153
-      version=$COLLECTOR_VERSION
-    fi
-
-    if [ -z "$version" ] ; then
-      version=$(latest_version)
-    fi
-
-    if [ -z "$version" ] ; then
-      error_exit "$LINENO" "Could not determine version to install"
-    fi
-
     if [ -z "$base_url" ] ; then
       base_url=$DOWNLOAD_BASE
     fi
 
-    collector_download_url="$base_url/v$version/${PACKAGE_NAME}_v${version}_linux_${os_arch}.${package_type}"
+    collector_download_url="$base_url/$version/${PACKAGE_NAME}_v${version}_linux_${os_arch}.${package_type}"
```

Comment:
Bucket directories do not have the `v` prefix.

<img width="834" height="588" alt="Screenshot 2025-07-19 at 3 48 38 PM" src="https://github.com/user-attachments/assets/10f40244-64ce-43ee-ac71-1783a102ef9a" />